pyrene/src/components/DateTimePicker/DateTimePicker.examples.tsx

```diff
@@ -0,0 +1,41 @@
+import { Example, StateProvider } from '../../examples/Example';
+import { DateTimePickerProps } from './DateTimePicker';
+
+export interface State {
+  value?: number | null
+}
+
+const DateTimePicker: Example<DateTimePickerProps, State> = {
+  props: {
+    onChange: (stateProvider: StateProvider<State>) => (value?: number | null) => { stateProvider.setState({ value }); },
+  },
+
+  examples: [
+    {
+      props: {
+        dateOnly: true,
+        minDateTime: 946684740000,
+        maxDateTime: 1809631860000,
```

Comment:
I agree this is perfect for readability but during the planning of product, it was said that it would be best to as much as possible stick to the use of UTC timestamps. As this would remove ambiguity and these would be the values stored in the database anyway.